src/main/g8/default.properties

```diff
@@ -1,3 +1,6 @@
-name=my-awesome-lambda
+name=My Awesome Lambda
 stack=mobile
```

Comment:
I think we should set all properties that need to be changed by the user to a universal thing, such as `CHANGE_ME` so they match the standard in my scala play giter8 template.